Source/WebCore/dom/Observable.cpp

```diff
@@ -122,6 +124,22 @@ Ref<Observable> Observable::drop(ScriptExecutionContext& context, uint64_t amoun
     return create(createSubscriberCallbackDrop(context, *this, amount));
 }
 
+Ref<Observable> Observable::inspect(ScriptExecutionContext& context, std::optional<InspectorUnion> inspectorUnion)
```

Comment:
`std::optional<InspectorUnion>&&`

Reply:
As mentioned [here](https://github.com/WebKit/WebKit/pull/37520#discussion_r1874197025) I was not able to get a compilation doing this.